frontend/openchat-client/src/events.ts

```diff
@@ -48,6 +48,12 @@ export class ChatUpdated extends Event {
     }
 }
 
+export class ChatsLoaded extends Event {
```

Comment:
I slightly wonder about the name `ChatsLoaded` just because it might not quite convey what it means. Specifically it misses that this is something that is happening periodically and also that it may not have loaded any chats. 

Maybe `ChatsSynchronised`?

Reply:
ChatsUpdated?